### ISSUE TITLE
Rename citizen-engagement-cards.html

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3756]
### Dependency
- #3757 
- #3738 
- #3802 

### Overview
We are going to rename the `citizen-engagement-cards.html` to `program-area-pages-cards.html` so that the name of the file reflects the purpose of the card better.

### Details
The cards will be used in the future on more than just the Citizen Engagement webpage [see Resources/Instructions 1]. The card will be used for future program area pages, such as an Environment webpage and a Social Safety Net webpage.

#### Developer
- [x] In your IDE on your computer, navigate to the `_includes` directory and find `citizen-engagement-cards.html`
  - [x] Rename `citizen-engagement-cards.html` to `program-area-pages-cards.html`
  - [x] In the file, add this comment at the top
    ```
    <!-- These are the cards found on a program area page, such as the cards on https://www.hackforla.org/citizen-engagement webpage. -->
    ```
- [x] In `pages/citizen-engagement.html`, change
```
{% include citizen-engagement-cards.html %}
```
to
```
{% include program-area-pages-cards.html %}
```
- [x] Using Docker and developer tools, check your changes 
  - [x] look good in desktop, tablet, and mobile views, 
  - [x] that nothing breaks, and
  - [x] it still functions and looks the same as the Citizen Engagement page on the live website
- [x] Once your pull request has been merged, 
  - [x] In the `Citizen Engagement page` wiki page [see Resources/Instructions 4], change the following line from:
    ```
      * [_includes/citizen-engagement-cards.html](https://github.com/hackforla/website/blob/gh-pages/_includes/citizen-engagement-cards.html): HTML file that provides the layout/format for each card on the Citizen Engagement page
    ```    
    to
    ```
      * [_includes/program-area-pages-cards.html](https://github.com/hackforla/website/blob/gh-pages/_includes/program-area-pages-cards.html): HTML file that provides the layout/format for each card on the Citizen Engagement page
        * This file used to be called `citizen-engagement-cards.html`.
    ```
#### Merge Team, Technical Lead, or Product Manager
- [x] Once the pull request associated with this issue is merged, go to issue #4159 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#4159) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.

### Resources/Instructions
1. [Citizen Engagement webpage](https://www.hackforla.org/citizen-engagement)
2. [_includes/citizen-engagement-cards.html](https://github.com/hackforla/website/blob/gh-pages/_includes/citizen-engagement-cards.html)
3. [pages/citizen-engagement.html](https://github.com/hackforla/website/blob/gh-pages/pages/citizen-engagement.html)
4. [Citizen Engagement page wiki page](https://github.com/hackforla/website/wiki/Citizen-Engagement-page)

**Comment by JessicaLucindaCheng** [comment]
### Issue writing ETA and availability
- Availability: 6 hours
- ETA: Fri, Dec 9, 2022

### To Dos
- [x] Add this issue to the dependency section in #3328
- [ ] Add Action Items
  - [ ] Rename citizen-engagement-cards.html to something relevant program-area-page-card-layout.html
  - [ ] Figure out where the name is calling it and change it there too

**Comment by JessicaLucindaCheng** [comment]
### Issue writing ETA and availability
- Availability: 6 hours
- ETA: Sun, Jan 22, 2023 

### To Dos
- [x] Add this issue to the dependency section in #3328
- [x] Add Action Items
  - [x] Rename citizen-engagement-cards.html to something relevant
  - [x] Figure out where the name is calling it and change it there too

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to add a milestone and prioritize it.

**Comment by jdingeman** [comment]
@JessicaLucindaCheng - thank you for writing this! Marking as ready for prioritization.

**Comment by arcan9** [comment]
Availability: 03/15 - 03/17 after 7pm PST
ETA: 03/17 evening
